client/client.go

```diff
@@ -117,6 +119,18 @@ func NewClient(c *Config) (*Client, error) {
 				tc.RootCAs = rootCAs
 			}
 		}
+		//Specify client cert and key pair
+		if c.TLS.MtlsCliCrt != "" && c.TLS.MtlsCliKey != "" {
+			tc.GetClientCertificate = func(info *tls.CertificateRequestInfo) (*tls.Certificate, error) {
```

Comment:
Instead of using `GetClientCertificate` (which is called repeatedly) you should set `tc.Certificates` as an array wth only `c`.

Reply:
Will try.